data/multus/001-rbac.yaml

```diff
@@ -19,23 +19,23 @@ rules:
       - update
 ---
 kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: multus
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: multus
 subjects:
-- kind: ServiceAccount
-  name: multus
-  namespace: {{ .Namespace }}
+  - kind: ServiceAccount
+    name: multus
+    namespace: '{{ .Namespace }}'
```

Comment:
Why do we need single quota, what namespace is failing ?

Reply:
since the string I want to change (`{{ .Namespace }}`) has spaces, I need to pass it with `""` all the way to the yq function. This results with the final yq to run like this:
```
__yq w -i ./ServiceAccount_multus.yaml metadata.namespace '{{ .Namespace }}'
```

This is why we get the `''`, but no matter what I tried I couldn't remove the `''` (because like I said, removing the `""` always breaks the argument to words and gives me this `namespace: '{{'`).

Do you have any workaround to this issue?

Reply:
the only thing I can come up with is to run `sed 's/'{{ .Namespace }}'/{{ .Namespace }}/g'` to the final yaml, or something of that sort, but it's ugly